### PR TITLE
fix-upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Sometimes package_check fails on upgrade check*

## Solution
- *Fix upgrade*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/wekan_ynh%20PR63%20(yalh76)/)

### CODE REVIEW

File: scripts/ynh_package_version [hunk 1]
```diff
@@ -15,6 +15,7 @@ abort_if_up_to_date () {
     version=$(read_json "/etc/yunohost/apps/$YNH_APP_INSTANCE_NAME/manifest.json" 'version' 2> /dev/null || echo '20160501-7')
     last_version=$(read_manifest 'version')
     if [ "${version}" = "${last_version}" ]; then
+        sleep 60
```
Comment (alexAubin) [inline scripts/ynh_package_version]: uuuuuh, ... 60, really o.O ? To ... do nothing O.ô ?

serveimage

Comment (yalh76) [reply scripts/ynh_package_version]: I know.. I'm trying to figure why I have an error "502 Bad gateway" at upgrade check on my own CI...
All other packages go well, even some package not working on DEV CI, work on my CI....

Comment (yalh76) [reply scripts/ynh_package_version]: And now there is the same error on CI-APPS: https://ci-apps.yunohost.org/ci/job/5412 :'/

**Comment by yalh76** [comment]
Seems now solved

**Pull request closed without merging**